core/controllers/reader_test.py

```diff
@@ -260,6 +260,13 @@ def test_get_exploration_pretests(self) -> None:
                 'cmd': story_domain.CMD_ADD_STORY_NODE,
                 'node_id': 'node_1',
                 'title': 'Title 1'
+            }), story_domain.StoryChange({
```

Comment:
Out of curiosity, why are we changing this test?

Reply:
Due to my changes, the story domain validation checks require the nodes to have linked exploration IDs, as opposed to checking for this at every query method/function. Because of this, the tests that try to add nodes without linking explorations to them will fail.